pkg/metrics/metrics.scss

```diff
@@ -184,18 +184,14 @@
   }
 
   // Don't wrap network in / out data
-  td[data-label="In"],
```

Comment:
As already requested by Matej and Garrett, please split this out into a separate commit.